frontend/public/components/utils/cog.tsx

```diff
@@ -107,12 +107,12 @@ export class Cog extends DropdownMixin {
       { isDisabled ?
         <Tooltip content="disabled">
           <div ref={this.dropdownElement} className={classNames('co-m-cog', {'co-m-cog--disabled' : isDisabled})} >
-            <span className={classNames('fa', 'fa-cog', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
+            <span role="button" tabIndex={0} className={classNames('fa', 'fa-ellipsis-v', 'co-m-cog__icon', {'co-m-cog__icon--disabled' : isDisabled})} aria-hidden="true"></span>
```

Comment:
This should really be a button, not a span, to be accessible. Adding the tab index and role doesn't totally fix it. You can't open it with the keyboard.

Reply:
If you want it to be a button, should I modify Dropdown so we can just use a Dropdown component rather than building something custom with DropdownMixin? Dropdown has support for keyboard navigation; DropdownMixin doesn't right now.